packages/handbook-v1/en/Basic Types.md

```diff
@@ -175,42 +175,79 @@ let colorName: string = Color[2];
 You might expect `Object` to play a similar role, as it does in other languages.
-However, variables of type `Object` only allow you to assign any value to them. You can't call arbitrary methods on them, even ones that actually exist:
+However, variables of type `Object` do not allow you to assign `null` or `undefined` when `--strictNullChecks` is specified.
 
 ```ts twoslash
-// @errors: 2339
-let notSure: any = 4;
-notSure.ifItExists(); // okay, ifItExists might exist at runtime
-notSure.toFixed(); // okay, toFixed exists (but the compiler doesn't check)
+// @errors: 2322
+const notSure: Object = null; // error, type 'null' is not assignable to type 'Object'
+```
+
+If you have a variable with an unknown type, you can narrow it to something more specific by doing `typeof` checks, comparison checks, or more advanced type guards that will be discussed in a later chapter:
 
-let prettySure: Object = 4;
-prettySure.toFixed(); // Error: Property 'toFixed' doesn't exist on type 'Object'.
+```ts twoslash
+// @errors: 2322 2322 2322
+declare const maybe: unknown;
+const aNumber: number = maybe; // error, 'maybe' could be a string, object, boolean, undefined, or an array
+if (maybe === true) {
+  const aBoolean: boolean = maybe; // ok
+  const aString: string = maybe; // error, 'maybe' is definitely a boolean, so definitely NOT a string
+}
+if (typeof maybe === "string") {
+  const aString: string = maybe; // ok
+  const aBoolean: boolean = maybe; // error, 'maybe' is definitely a string, so definitely NOT a boolean
+}
+```
+
+# Any
+
+In some situations, not all type information is available or it's declaration would take an inappropriate amount of effort.
+These may occur for values from code that has been written without TypeScript or a 3rd party library.
+In these cases, we might want to opt-out of type checking.
+To do so, we label these values with the `any` type:
+
+```ts twoslash
+declare function getValue(key: string): any;
+const str: string = getValue("myString"); // ok, return value of 'getValue' is not checked
+```
+
+The `any` type is a powerful way to work with existing JavaScript, allowing you to gradually opt-in and opt-out of type checking during compilation.
+Just like `unknown`, variables of type `any` allow you to assign any value to them.
+Unlike `unknown`, TypeScript let you can call arbitrary methods on them, even ones that don't exist:
+
+```ts twoslash
+// @errors: 2571
+let looselyTyped: any = 4;
+looselyTyped.ifItExists(); // okay, ifItExists might exist at runtime
+looselyTyped.toFixed(); // okay, toFixed exists (but the compiler doesn't check)
+
+let strictlyTyped: unknown = 4;
+strictlyTyped.toFixed(); // Error: Property 'toFixed' doesn't exist on type 'unknown'.
 ```
 
 The `any` type is also handy if you know some part of the type, but perhaps not all of it.
 For example, you may have an array but the array has a mix of different types:
 
 ```ts twoslash
 let list: any[] = [1, true, "free"];
-
 list[1] = 100;
 ```
```

Comment:
I recommend removing this section, when this was originally written `any` was a solution to this problem, but the language now has `tuples` and union types which are a much better solution to this problem.

Reply:
Yeah, I couldn't think of a good example like this which doesn't rely on using an interface/type, so I dropped it

Reply:
Agree. I think the paragraph is still true in rare cases but that is nothing a new user should worry about and probably to complex to draft here.